### ISSUE TITLE
[环境测试]建议添加requirements.txt

### ISSUE THREAD

**Issue by hhhllll888** [hzy46/fast-neural-style-tensorflow#72]
测试，在以下环境可以使用
~~~
absl-py==1.4.0
astunparse==1.6.3
cachetools==5.3.1
certifi==2023.7.22
charset-normalizer==3.2.0
distlib==0.3.7
filelock==3.12.4
flatbuffers==23.5.26
gast==0.4.0
google-auth==2.23.0
google-auth-oauthlib==1.0.0
google-pasta==0.2.0
grpcio==1.58.0
h5py==3.9.0
idna==3.4
keras==2.13.1
libclang==16.0.6
Markdown==3.4.4
MarkupSafe==2.1.3
numpy==1.24.3
oauthlib==3.2.2
opt-einsum==3.3.0
packaging==23.1
platformdirs==3.10.0
protobuf==4.24.3
pyasn1==0.5.0
pyasn1-modules==0.3.0
PyYAML==6.0.1
requests==2.31.0
requests-oauthlib==1.3.1
rsa==4.9
six==1.16.0
tensorboard==2.13.0
tensorboard-data-server==0.7.1
tensorflow==2.13.0
tensorflow-estimator==2.13.0
tensorflow-intel==2.13.0
tensorflow-io-gcs-filesystem==0.31.0
termcolor==2.3.0
tf-slim==1.1.0
typing_extensions==4.5.0
urllib3==1.26.16
virtualenv==20.24.5
Werkzeug==2.3.7
wrapt==1.15.0
~~~
